### PR TITLE
CNV-19778: Update links on Getting Started

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.11

Issue: [CNV-19778](https://issues.redhat.com/browse/CNV-19778)

Link to docs preview: [Getting started with OpenShift Virtualization](http://file.tlv.redhat.com/apinnick/CNV-19778-update-getting-started/virt/virt-learn-more-about-openshift-virtualization.html)

### CODE REVIEW

File: virt/virt-learn-more-about-openshift-virtualization.adoc [hunk 1]
```diff
@@ -16,35 +16,38 @@ Cluster configuration procedures require `cluster-admin` privileges.
 [id="virt-getting-started-before-you-begin"]
 == Before you begin
 
-* Review the xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[installation requirements].
-* Review the xref:../virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc#virt-features-for-storage[storage features] required for cloning, snapshots, and live migration. See xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider] for details.
+* Review xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Preparing your cluster for {VirtProductName}].
```
Comment (ousleyp) [inline virt/virt-learn-more-about-openshift-virtualization.adoc:19]: I might rephrase this to make it more like the other bullets:

```suggestion
* Learn how to xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[prepare your cluster for {VirtProductName}].
```

File: virt/virt-learn-more-about-openshift-virtualization.adoc [hunk 2]
```diff
@@ -54,23 +57,28 @@ Configure additional networks::
...
 * xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator]
 * xref:../virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc#virt-specifying-nodes-for-vms[Specifying nodes for virtual machines]
-* xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration]
+
+Getting started::
 * xref:../virt/vm_templates/virt-creating-vm-template.adoc#virt-creating-vm-template[Virtual machine templates]
-* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Configuring local storage]
 
+Next steps::
+* xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration]
```
Comment (ousleyp) [inline virt/virt-learn-more-about-openshift-virtualization.adoc:83]: I recall that there was an issue with Jupiter and using any formatting other than unordered lists in an additional resources section. [The contribution guide still seems to indicate this.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) For this page, it might make sense to add the links to the existing sections, maybe as sub-bullets to differentiate them from the more basic resources?

**Review by ousleyp — COMMENTED** [review]
This mostly LGTM, but there's a potential issue with the additional resources section formatting + one suggestion. Thanks!

File: virt/virt-learn-more-about-openshift-virtualization.adoc (hunk 1 above)
Comment (apinnick) [reply virt/virt-learn-more-about-openshift-virtualization.adoc:19]: Thanks but I might rephase this to make it stronger. I don't want them to "learn" how to prepare their cluster. I want them to do it! :-)

File: virt/virt-learn-more-about-openshift-virtualization.adoc (hunk 2 above)
Comment (apinnick) [reply virt/virt-learn-more-about-openshift-virtualization.adoc:83]: Sure, that works for me!
